scripts/version updates/README.md

```diff
@@ -0,0 +1,87 @@
+# MeiliSearch version Updates
+
+This scripts lets you update your MeiliSearch instance to a specific version, including `rc's`.
+
+## Requirements
+
+### 1. Systemctl 
+MeiliSearch must be running using `systemctl` and thus an environment in which `systemctl` exists.
+
+### 2. data.ms path
+
+MeiliSearch `data.ms` must me stored at the following address: `/var/lib/meilisearch/data.ms`.
+  To ensure this is the case, MeiliSearch should have started with the following flags:`--db-path /var/lib/meilisearch/data.ms`
+
+You can check the information by looking at the file located here `cat /etc/systemd/system/meilisearch.service`.
+You should find a line with the specific command used.
+
+```bash
+ExecStart=/usr/bin/meilisearch --db-path /var/lib/meilisearch/data.ms --env production
+```
+
+### 3. A Running MeiliSearch instance
+
+A MeiliSearch instance should be running before launching the script. This can be checked using the following command: 
+
+```bash
+systemctl status meilisearch
+```
+
+## Usage
+
+⚠️ You may lose your data using this script, if you have no easy way to re-index it, we suggest manually [creating you own dump](https://docs.meilisearch.com/reference/features/dumps.html#creating-a-dump). 
```

Comment:
```suggestion
⚠️ In case of failure during the execution of the script and for security purposes, we strongly recommend [creating manually your own dump](https://docs.meilisearch.com/reference/features/dumps.html#creating-a-dump).
```

If the migration fails their is a fallback that will put back the old version of MeiliSearch with its original data.ms.
Is this suggestion okey for you?

Reply:
Yes!!! :D